pieria/style.css

```diff
@@ -0,0 +1,66 @@
+/*
+Theme Name: Pieria
+Theme URI: https://github.com/wordpress/pieria/
+Author: the WordPress team
+Author URI: https://wordpress.org/themes/
+Description: Pieria is a theme focused on typography — essentially following a low-level styling approach. Its headings, paragraphs, links, and navigation have the same appearance, with everything sitting quietly into simple columns that split navigation, blog posts, and a content block. A soft design to make your content sparkle.
+Requires at least: 5.8
+Tested up to: 5.9
+Requires PHP: 5.7
+Version: 0.0.1
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Template: 
+Text Domain: pieria
+Tags: blog, one-column, wide-blocks, block-patterns, custom-colors, custom-logo, custom-menu, editor-style, featured-images, full-site-editing, rtl-language-support, theme-options, threaded-comments, translation-ready, blog, simple-format, low-formatting, blog-posts, serif, serifs, simple-blog, simple-blogging
+*//*
+ * Font smoothing
+ * https://github.com/WordPress/gutenberg/issues/35934
+ */
+body {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+}
```

Comment:
We likely don't need to adjust the font-smoothing, this is an old CSS snippet that Block Canvas used to include.